.eslintrc.js

```diff
@@ -3,35 +3,20 @@ module.exports = {
     es6: true,
     node: true,
   },
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint", "jest"],
   extends: [
-    'airbnb-base',
```

Comment:
I don't feel strongly about any of these options if we want to keep the existing ones. Just reading up on the VSCode extension API and thought I could contribute some of the suggestions back.

The main goal is to enable eslint in the editor when working on this extension